### PR TITLE
make benchmark_model for tflite build

### PR DESCRIPTION
**Pull request by freedomtan**
benchmark_model doesn't build for desktop as reported at https://github.com/tensorflow/tensorflow/issues/20313

### CODE REVIEW

**Comment by andrewginns** [comment]
This fix worked for me
Ubuntu 16.04
Bazel 0.10.0

File: tensorflow/contrib/lite/tools/benchmark/benchmark_params.h [hunk 1]
```diff
@@ -46,11 +46,11 @@ class BenchmarkParam {
   }
   virtual ~BenchmarkParam() {}
   BenchmarkParam(ParamType type) : type_(type) {}
+  template <typename T>
```
Comment (shashishekhar) [inline tensorflow/contrib/lite/tools/benchmark/benchmark_params.h]: Can you move this to protected section.

**Review by shashishekhar — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/lite/tools/benchmark/benchmark_params.h (hunk 1 above)
Comment (shashishekhar) [reply tensorflow/contrib/lite/tools/benchmark/benchmark_params.h]: Thank you for the patch!

Comment (freedomtan) [reply tensorflow/contrib/lite/tools/benchmark/benchmark_params.h]: Done. Thanks for the review.

**Review by shashishekhar — APPROVED** [review]

**Comment by shashishekhar** [comment]
Unfortunately merging this will have to wait, see 
https://groups.google.com/a/tensorflow.org/forum/#!topic/announce/6ojJ7ykrmZ0

**Pull request merged**